MsgStripe stripe
int32 number
int32 locked
